stable/presto/templates/deployment-coordinator.yaml

```diff
@@ -25,6 +25,15 @@ spec:
         - name: config-volume
           configMap:
             name: {{ template "presto.coordinator" . }}
+        {{- if .Values.server.catalog.secretName }}
+        - name: catalog-volume
+          secret:
+            secretName: {{ .Values.server.catalog.secretName }}
+        {{- else if .Values.server.catalog.data | quote }}
```

Comment:
Why do you need this else condition here. Under what circumstance would the secret location change or not be what is specified in the first conditional.

Reply:
it provides the options to configures connectors via either k8s secret or configmap.